pkgs/development/python-modules/kestra/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+}:
+buildPythonPackage rec {
+  pname = "kestra";
+  version = "0.20.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-j2ASwbleCsMrqhefhdBwXfp6JA+tBr1CX/nIKO4sWZI=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    requests
+  ];
+
+  pythonImportsCheck = [
+    "kestra"
+  ];
+
+  meta = {
+    description = "Infinitely scalable orchestration and scheduling platform, creating, running, scheduling, and monitoring millions of complex pipelines.";
```

Comment:
```suggestion
    description = "Scalable orchestration and scheduling platform for pipelines";
```

Reply:
In that case you at least need to drop the final period.

Reply:
Sure. I'll fix it alongside the pytest check 👍 .